dashboard/test/factories/pd_factories.rb

```diff
@@ -1646,5 +1666,28 @@
       },
       "k5_facilitator_did_well":"things done well"
     }'
+    end
+
+    trait :answers_high do
+      answers '{
+      "facilitatorId": 1,
+      "facilitatorName": "Facilitator1",
+      "facilitator_effectiveness":{
+        "demonstrated_knowledge":"7",
+        "built_equitable":"7",
+        "on_track":"7",
+        "productive_discussions":"7",
+        "ways_equitable":"7",
+        "healthy_relationship":"7"
+      },
+      "k5_facilitator_did_well":"things done well"
+    }'
+    end
+  end
+
+  trait :foorm_submission_metadata do
+    form_version 0
+    created_at "2020-03-30 21:58:28"
+    updated_at "2020-03-30 21:58:28"
```

Comment:
How important is it that these get locked to a particular value? Can we remove these two lines and let default ActiveRecord behaviors fill this in?

Reply:
we can do the default for sure, I will update